src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py

```diff
@@ -421,8 +407,16 @@ def ensure_container_insights_for_monitoring(
             if data_collection_settings is not None:
                 dataCollectionSettings = _get_data_collection_settings(data_collection_settings)
                 validate_data_collection_settings(dataCollectionSettings)
+                dataCollectionSettings.setdefault("enableContainerLogV2", True)
```

Comment:
what happens if customer also applies a configmap with schema set to v1?

Reply:
that is a good question, thanks for catching this scenario, in this case in order to set it to v1, for legacy user they don't need to since there is no DCR; for MSI cluster, since we default verything to v2, they have to set the DCR to false as well. cc @ganga1980

Reply:
This change shouldnt impact non-MSI/legacy container insights auth clusters (new or existing). For MSI clusters (new or existing),  with this CLI version, default to ContainerLogV2 and customer can't configure to v1.  For customers to use v1 schema, customer needs to use the previous version of the CLI.  @wanlonghenry , lets discuss with @austonli  on this and see if we are good make this change, and discuss if we need any doc updates before or after the change.

Reply:
for MSI clusters they can still go to v1 mode as long as they configured the DCR, will schedule a meeting with Auston for further discussion